src/analysis/locally_convex/with_seminorms.lean

```diff
@@ -23,6 +23,16 @@ bounded by a finite number of seminorms in `E`.
 * `seminorm_family.to_locally_convex_space`: A space equipped with a family of seminorms is locally
 convex.
 
+## Relation between boundedness and continuity
```

Comment:
I'm not sure I understand why you classify these lemmas as "relations between boundedness and continuity", could you expand on that (both here and in the dosctring)?

Reply:
I've changed it, the title was not really good and I've moved the direct criterion into that section